pkg/lifecycle/versions.rules.go

```diff
@@ -125,3 +136,12 @@ func (vr *VersionRules) CheckChartVersionForLifecycle(chartVersion string) bool
 	}
 	return false
 }
+
+// CheckChartVersionToRelease will return if the current versyion being analyzed is the one to be released or not
+func (vr *VersionRules) CheckChartVersionToRelease(chartVersion string) bool {
+	chartVersionInt, _ := strconv.Atoi(strings.Split(chartVersion, ".")[0])
```

Comment:
Is there ever a chance this could error?

Reply:
This should never fail, but I am adding error handling here; we never know what might happen.